hs/t/n/mint-fail1.txt

```diff
@@ -9,23 +9,133 @@ Verification failed:
 
   // Violation Witness
 
-  const v257 = protect<Object({"amt": UInt, "doEarlyTransfer": Bool, "metadata": Bytes(32), "name": Bytes(32), "supply": UInt, "symbol": Bytes(8), "url": Bytes(96)})>("Alice".interact.getParams());
+  const UInt.max = 1;
+
+  const v314 = protect<Object({"amt": UInt, "doEarlyTransfer": Bool, "metadata": Bytes(32), "name": Bytes(32), "supply": UInt, "symbol": Bytes(8), "url": Bytes(96)})>("Alice".interact.getParams());
   //    ^ could = {amt: 0, doEarlyTransfer: false, metadata: "Bytes!val!0", name: "Bytes!val!1", supply: 1, symbol: "Bytes!val!2", url: "Bytes!val!3" }
   //      from: ./mint-fail1.rsh:24:104:application
 
   // Theorem Formalization
 
-  const v362 = 0 == v257.supply;
+  const v307 = [0, 0, false ];
```

Comment:
This is really confusing... is it plausible to insert in some labels on the variables in your variable making function?